insolar/payload/payload.proto

```diff
@@ -372,3 +372,15 @@ message UpdateJet {
     bytes Pulse = 20 [(gogoproto.customtype) = "github.com/insolar/insolar/insolar.PulseNumber", (gogoproto.nullable) = false];
     bytes JetID = 21 [(gogoproto.customtype) = "github.com/insolar/insolar/insolar.JetID", (gogoproto.nullable) = false];
 }
+
+message GetPulse {
+    uint32 Polymorph = 16;
+
+    bytes PulseNumber = 20 [(gogoproto.customtype) = "github.com/insolar/insolar/insolar.PulseNumber", (gogoproto.nullable) = false];
+}
+
+message Pulse {
+    uint32 Polymorph = 16;
+
+    pulse.PulseProto Pulse = 24 [(gogoproto.nullable) = false];
```

Comment:
Why pulse field number is 24 and not 20?

Reply:
changed